dali/python/nvidia/dali/auto_aug/rand_augment.py

```diff
@@ -173,13 +173,13 @@ def apply_rand_augment(augmentations: List[_Augmentation], sample: _DataNode, n:
     _forbid_unused_kwargs(augmentations, kwargs, 'apply_rand_augment')
     for level_idx in range(n):
         level_mag_bin = mag_bin if not use_signed_magnitudes or n == 1 else mag_bin[level_idx]
-        op_kwargs = dict(sample=sample, magnitude_bin=level_mag_bin,
+        op_kwargs = dict(data=data, magnitude_bin=level_mag_bin,
                          num_magnitude_bins=num_magnitude_bins, **kwargs)
         level_op_idx = op_idx if n == 1 else op_idx[level_idx]
-        sample = _pretty_select(augmentations, level_op_idx, op_kwargs,
+        data = _pretty_select(augmentations, level_op_idx, op_kwargs,
                                 auto_aug_name='apply_rand_augment',
                                 ref_suite_name='get_rand_augment_suite')
```

Comment:
The two lines seem to be overindented now.

Reply:
This is why we should have autoformatting as an option.